src/lib/Scenes/City/City.tsx

```diff
@@ -206,3 +219,18 @@ const Handle = styled.View`
   border-radius: 2.5px;
   background-color: ${color("black30")};
 `
+
+const ErrorScreen: React.SFC<{ relayErrorState: RelayErrorState }> = ({ relayErrorState: { retry, isRetrying } }) => {
+  return (
+    <Box py={2}>
+      <Sans size="3t" textAlign="center" mx={2}>
+        We are having trouble loading content right now, please try again later.
```

Comment:
I dunno the spec, but "the content" feels a bit meh - "the city guide" feels a bit better

Reply:
Yeah, let's get feedback on that in peace room tomorrow.